libevmcore/Instruction.h

```diff
@@ -184,6 +175,17 @@ enum class Instruction: uint8_t
 	JUMPC,              ///< alter the program counter - pre-verified
 	JUMPCI,             ///< conditionally alter the program counter - pre-verified
 
+	JUMPTO = 0xb0,      ///< alter the program counter to a jumpdest
```

Comment:
The comments for JUMPTO & JUMPV are now equal (as are JUMPSUB & JUMSUBV). Some distinguishing note would be helpful